### PR TITLE
Kolibri Windows installer signing documentation.

### LINKED ISSUE #73: Document the Windows installer signing at the buildkite.
Create a documentation on how to setup the Windows installer signing at the buildkite from scratch.

### PR DESCRIPTION
**Pull request by mrpau-richard**
This fixes #73 

A documentation on how to setup the Windows installer signing at the google cloud and buildkite from scratch.

### CODE REVIEW

**Comment by indirectlylit** [comment]
This is great! thanks for working on it.

@radinamatic would you mind going through these steps as written and giving any feedback on the docs?

@mrpau-richard looks like there's a space in the directory name?

image

**Comment by mrpau-richard** [comment]
I already removed it @indirectlylit . I did not notice that extra space in the file name.

File: windows/kolibri-signing-doc/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+You'll need accounts with access to Learning Equality's organization on these services:
+
+* [Buildkite](https://buildkite.com) account.
+* [Google cloud](https://cloud.google.com/) account.
+
+
+## Setting up the Windows signing pipeline
+
+* Create a Windows VM instance on google cloud
+	- Log in to [Google cloud](https://cloud.google.com/)
```
Comment (radinamatic) [inline windows/kolibri-signing-doc/README.md]: Capitalization: `Google Cloud` here and in the line above.

File: windows/kolibri-signing-doc/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	- Read the [buildkite pipeline](https://buildkite.com/docs/pipelines) documentation
+	- Use `https://github.com/learningequality/kolibri.git` as the Github repository
+
+* Set up buildkite environment variables
+	- In buildkite, navigate to Kolibri's `Pipeline Settings` > `Environment variables`
+	- Add the environment variable `WINDOWS_SIGN_SCRIPT_PATH=C:\xxxx\sign_script.bat`
+	- Replace `xxxx` above so the the `sign_script.bat` file path is correct
+
+* Build and sign a Kolibri Windows installer
+	- Navigate to the kolibri pipeline, and click `New Build`
```
Comment (radinamatic) [inline windows/kolibri-signing-doc/README.md]: Capitalize `Kolibri`.

File: windows/kolibri-signing-doc/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+You'll need accounts with access to Learning Equality's organization on these services:
+
+* [Buildkite](https://buildkite.com) account.
+* [Google Cloud](https://cloud.google.com/) account.
+
+
+## Setting up the Windows signing pipeline
+
+* Create a Windows VM instance on google Cloud
```
Comment (radinamatic) [inline windows/kolibri-signing-doc/README.md]: **_G_**oogle!

**Pull request merged**